robosuite/wrappers/gym_wrapper.py

```diff
@@ -83,15 +84,20 @@ def _flatten_obs(self, obs_dict, verbose=False):
                 ob_lst.append(np.array(obs_dict[key]).flatten())
         return np.concatenate(ob_lst)
 
-    def reset(self):
+    def reset(self, seed=None, options=None):
         """
-        Extends env reset method to return flattened observation instead of normal OrderedDict.
+        Extends env reset method to return flattened observation instead of normal OrderedDict and optionally resets seed
 
         Returns:
             np.array: Flattened environment observation space after reset occurs
         """
+        if seed is not None:
+            if isinstance(seed, int):
+                np.random.seed(seed)
+            else:
+                TypeError("Seed must be an integer type!")
```

Comment:
you need a `raise` statement here right? After that fix, I think we can merge

Reply:
Sorry, missed that! Should be fixed now